### PR TITLE
Quanify

### PR DESCRIPTION
**Pull request by LlamaOnDrugs**
Will need to :
update bitcoin.cpp (305) with correct DNS url.
update main.cpp (430/431) with correct DNS urls.
adjust main.cpp (440) with seed node IP addresses if possible for initial start.

### CODE REVIEW

File: db.h [hunk 1]
```diff
@@ -12,11 +12,11 @@
 
 #define MIN_RETRY 1000
 
-#define REQUIRE_VERSION 70215
+#define REQUIRE_VERSION 70214
 
 static inline int GetRequireHeight(const bool testnet = fTestNet)
 {
-    return testnet ? 40000 : 1080000;
+    return testnet ? 0 : 25000;
```
Comment (akshaynexus) [inline db.h]: set the current block num for finding up to date seeders

Comment (akshaynexus) [reply db.h]: ie in the place of 25k blocks put the current block num

Comment (LlamaOnDrugs) [reply db.h]: The seeder can't find out the current block number, without hooking into a locally running daemon which it currently doesn't need to.  I set it to 25,000 but can set it higher for sure.

Comment (LlamaOnDrugs) [reply db.h]: updated to 72698

**Review by akshaynexus — APPROVED** [review]

**Pull request merged**